src/main/scala/exercise4/Task2.scala

```diff
@@ -0,0 +1,15 @@
+package exercise4
+
+import zio.{ExitCode, URIO, ZIO}
+
+object Task2 extends zio.App {
+
+  val program: ZIO[Any, String, Unit] = {
+    for {
+      task <- (ZIO.interrupt zipPar ZIO.fail("Error")).fork
```

Comment:
It does produce the same stack trace, however`ZIO.interrupt` is a bit weird, since it just starts a zio and immediatelly interrupts it.
Something like ```ZIO.succeed(2).forever``` would be more sensible. That would get interrupted due to `zipPar`, since `ZIO.fail("Error")` failed.